pkg/metrics/collector_test.go

```diff
@@ -450,20 +462,20 @@ var _ = Describe("ResourcesMetricsCollector", func() {
 			const expected = `
         # HELP trivy_resource_configaudits Number of failing resource configuration auditing checks
         # TYPE trivy_resource_configaudits gauge
-        trivy_resource_configaudits{name="configmap-test",namespace="some-ns",resource_kind="ConfigMap",resource_name="test",severity="Critical"} 0
```

Comment:
`summary report` test should remain , info test should be separate test

Reply:
@chen-keinan I'm not to clear on what changes to make here